src/components/Select/Select.js

```diff
@@ -1,9 +1,17 @@
 import React from 'react'
 import LibSelect from 'react-virtualized-select'
-import './Select.module.scss'
+import cx from 'classnames'
+import './Select.scss'
+import styles from './Select.module.scss'
 
-const Select = ({ ...props }) => {
-  return <LibSelect {...props} optionHeight={32} />
+const Select = ({ className, topDropdown, ...props }) => {
```

Comment:
One more select in our source. Can we use wrapper in san-ui? @DmitriVanGuard

Reply:
@Partysun Sure! I will move it out today